crates/sdk/src/masp.rs

```diff
@@ -2478,8 +2478,7 @@ impl<U: ShieldedUtils + MaybeSend + MaybeSync> ShieldedContext<U> {
                         let mut transfer = None;
                         extract_payload(tx, &mut wrapper, &mut transfer)?;
                         // Epoch data is not needed for transparent transactions
-                        let epoch =
-                            wrapper.map(|x| x.epoch).unwrap_or_default();
+                        let epoch = Epoch(0);
```

Comment:
I believe in here we should extract the `Epoch` at which the transaction was applied

Reply:
Ah nevermind, ignore my last comment